smac/runhistory/runhistory2epm.py

```diff
@@ -88,7 +88,7 @@ def __init__(self, scenario: Scenario, num_params: int,
 
         # Fill with some default values
         if rng is None:
-            self.rng = np.random.RandomState()
+            self.rng = np.random.RandomState(seed=1)
```

Comment:
Why is this a fixed seed in contrast to the other changed lines?

Reply:
@aaronki: The link to the script you provided doesn't work for me. Could you have a look at that?

Reply:
A fixed seed is now the default behaviour when no seed was given (see comment from @mlindauer  above: »... I would expect that the initialize our own rng in this case ...«